argo-cd-apps/base/host/konflux-workspaces/konflux-workspaces.yaml

```diff
@@ -0,0 +1,41 @@
+apiVersion: argoproj.io/v1alpha1
+kind: ApplicationSet
+metadata:
+  name: konflux-workspaces
```

Comment:
This is very subjective comment: all other components do not use "konflux" prefix in the applicationset name as well as the folder in the components folder. I am OK if you want to keep it like that but this change is adding the workspaces component/service to konflux so konflux prefix is redundant to some extent. 

should we name this `workspaces`, `workspaces-services` or `workspaces-controller` or ...  to be consistent with other components/services. here are the names of all other components:
```
application-api
authentication
backup
build-service
build-templates
ci-helper-app
cluster-as-a-service
cluster-secret-store
cluster-secret-store-rh
dev-sso
dora-metrics
enterprise-contract
external-secrets-operator
gitops
has
image-controller
integration
internal-services
jvm-build-service
keycloak
konflux-ci
konflux-workspaces
mintmaker
monitoring
multi-platform-controller
openshift-gitops
perf-team-prometheus-reader
pipeline-service
project-controller
quality-dashboard
release
remote-secret-controller
repository-validator
sandbox
segment-bridge
smee
smee-client
spacerequest-cleaner
spi
spi-vault
sprayproxy
tekton-ci
tracing
ui
```

Reply:
I think workspaces is good enough